src/aitsi/m3spin/pkb/interfaces/Follows.java

```diff
@@ -11,7 +11,7 @@ public interface Follows {
 
     List<TNode> getFollows$(TNode n);
```

Comment:
Skoro relacja Follows dotyczy wyłącznie instrukcji, to zastanawiam się, czy nie zastąpić wszystkich typów `TNode` w tym interfejsie typem `Statement`

Reply:
Poprawion